### PR TITLE
fix #177 - add TypeScript type `NumberType` for `getNumberType` and `…

### LINKED ISSUE #177: TypeScript type definitions for return types of getNumberType and getNumberTypeCustom
Add TypeScript [String Literal Type](https://www.typescriptlang.org/docs/handbook/advanced-types.html) `NumberType` to indicate valid return values of `getNumberType` and `getNumberTypeCustom`

### PR DESCRIPTION
**Pull request by atsu85**
…getNumberTypeCustom`

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -1,6 +1,7 @@
...
 
 export type NumberFormat = 'National' | 'International' | 'E.164' | 'RFC3966';
+export type NumberType = undefined | 'PREMIUM_RATE' | 'TOLL_FREE' | 'SHARED_COST' | 'VOIP' | 'PERSONAL_NUMBER' | 'PAGER' | 'UAN' | 'VOICEMAIL' | 'FIXED_LINE_OR_MOBILE' | 'FIXED_LINE_OR_MOBILE' | 'FIXED_LINE' | 'MOBILE';
```
Comment (catamphetamine) [inline index.d.ts]: `FIXED_LINE_OR_MOBILE` is duplicated

Comment (atsu85) [reply index.d.ts]: oh, just copied return values from the implementation - good catch 👍

Comment (atsu85) [reply index.d.ts]: fixed (amended previous commit)

**Comment by atsu85** [comment]
Thanks for merging, could You publish it also?

**Comment by catamphetamine** [comment]
@atsu85 Released `libphonenumber-js@1.0.15`

**Pull request merged**